handler/list.go

```diff
@@ -0,0 +1,34 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// ListSession godoc
+// @Summary Lists all active session.
+// @Tags session
+// @Produce  application/json
+// @Success 200 {array} api.Session
+// @Failure 400 {object} api.Error
+// @Failure 401 {object} api.Error
+// @Failure 500 {object} api.Error
+// @Router /session [get]
+func (h *Handler) ListSession(rw http.ResponseWriter, r *http.Request) {
+	h.lock.Lock()
```

Comment:
use `defer h.lock.Unlock()` otherwise is a case of some panic the lock won't be released.